### DISCUSSION TITLE
Feature Request: Ability to show games on my Steam Wishlist

### DISCUSSION THREAD

**Discussion by bbennett2** [Raicuparta/rai-pal#158] [Ideas]
no clue if this is possible, but it would sure help me sort through my wishlist to see what i'm going to be purchasing next.  anything made in unreal that could have UEVR support is an immediate bump up the list for me!

**Comment by Raicuparta** [comment]
I don't think this will be in Rai Pal any time soon. Maybe something that can be accomplished on steamdb's website?
